public/app/features/alerting/unified/Insights.tsx

```diff
@@ -0,0 +1,80 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { EmbeddedScene, PanelBuilders, SceneFlexItem, SceneFlexLayout, SceneQueryRunner } from '@grafana/scenes';
+import { useStyles2 } from '@grafana/ui';
+
+function getScene() {
+  const queryRunner1 = new SceneQueryRunner({
+    datasource: {
+      type: 'loki',
+      //since all cloud instances have a provisioned alert state history data source,
+      //we should be able to use its uid here
+      uid: 'CKA1_2hVz',
```

Comment:
Do you think this is a good idea? do we have a unique uid for this data source or how can we obtain it?